lib/omniauth/strategies/oidc/back_channel_logout_request.rb

```diff
@@ -0,0 +1,81 @@
+module OmniAuth
+  module Strategies
+    class Authentiq
+      class BackChannelLogoutRequest
+
+        def initialize(strategy, request)
+          @strategy, @request = strategy, request
+        end
+
+        def call(options = {})
+          @options = options
+
+          begin
+            # for
+            # sign_out_callback.call(*back_channel_logout_request)
+            # to actually happen
+            # a proc must be set in the devise.rb initializer of the rails app
+            result = sign_out_callback.call(*back_channel_logout_request)
+          rescue
+            return backchannel_logout_response(400, ['Bad request']).finish
+          else
+            if result
+              result = backchannel_logout_response(200, ['OK'])
+            else
+              result = backchannel_logout_response(501, ['Not implemented'])
+            end
+          ensure
+            return unless result
+            return result.finish
+          end
+        end
+
+        private
+
+        def back_channel_logout_request
```

Comment:
Again ruby naming convention naming since on spec it is called  Back-Channel Logout